db/migrate/20240320112853_create_task_items.rb

```diff
@@ -4,6 +4,7 @@ class CreateTaskItems < ActiveRecord::Migration[7.1]
   def change
     create_table :task_items do |t|
       t.string :name, null: false
+      t.boolean :completed, null: false, default: false
```

Comment:
@lazaronixon, why is adding a new column necessary? The task item is incomplete if a completed_at is null.